code/citadel/organs/genitals_sprite_accessories.dm

```diff
@@ -27,15 +27,57 @@
 	icon_state = "tapered"
 	name = "Tapered"
 
+//Vaginas
+/datum/sprite_accessory/vagina
+	icon = 'code/citadel/icons/vagina_onmob.dmi'
+	icon_state = null
+	name = "vagina"
+	gender_specific = 0
+	color_src = "vag_color"
+	locked = 0
+
+/datum/sprite_accessory/vagina/human
+	icon_state = "human"
+	name = "Human"
+
+/datum/sprite_accessory/vagina/gaping
+	icon_state = "gaping"
+	name = "Gaping"
+
+//I think this one is intended for arousal... but it would only work for human vaginas in this case...
+//So I'm leaving it as a selectable option until we can make/need variants for the sprites to change based on arousal.
+/datum/sprite_accessory/vagina/dripping
+	icon_state = "dripping"
+	name = "Dripping"
+
+/datum/sprite_accessory/vagina/tentacles
+	icon_state = "tentacle"
+	name = "Tentacle"
+
+/datum/sprite_accessory/vagina/dentata
+	icon_state = "dentata"
+	name = "Dentata"
+
+/datum/sprite_accessory/vagina/hairy
+	icon_state = "hairy"
+	name = "Hairy"
```

Comment:
My worry with that is that it might be incompatible with the other shapes? Logically, you'd think you could have it on top of the shape, but some of them don't look like they lend themselves to it easily.
I imagine this wouldn't look very good overlayed on top of, say dentata or tentacles, since it's drawn at the same location as the other ones.
In fact, the fact that it's drawn exactly where the others are, seemed to me that the sprite isn't "hairy" as much as it looks "furry"? ((I admit it's hard to tell what the intention is, with the size of the sprites, but I know at least the coords of the sprite pixels means it will be right on top of the other ones, which I thought would interfere with some))